e2e/config/vhd.go

```diff
@@ -8,57 +8,81 @@ import (
 	"testing"
 
 	"github.com/Azure/agentbaker/pkg/agent/datamodel"
-	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
-	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
-	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v6"
 )
 
 const (
 	noSelectionTagName = "abe2e-ignore"
 )
 
+var (
+	linuxGallery = &Gallery{
+		SubscriptionID: Config.GallerySubscriptionID,
+		ResourceGroup:  Config.GalleryResourceGroup,
+		Name:           Config.GalleryName,
+	}
+
+	windowsGallery = &Gallery{
```

Comment:
can we avoid hardcoding ? and provide distinct configuration to allow switching Gallery dynamically,

Reply:
Generally, specifically for testing I find it's easier to define configuration inside tests (unless we want to have different contexts for some reason).

It's just easier to track and modify. An alternative typically requires more jumping through the code, then to the YAML, then trying to find the definition somewhere inside ADO UI.

But it looks like someone added the logic for Linux while I've been working on it. So, I guess there is a reason / migration work going on.